### PR TITLE
Add option to specify custom active tab button style

### PR DESCRIPTION
**Pull request by chrisza4**
### Motivation

Right now in each tab buttons, difference between active tab button and non-active tab button is background color, tint color and opacity.

Only way to have other custom styles for active tab button (For example, highlighted top border) in current API is to create whole custom tab, which I believe it is overkill.

I think it would be easier for this simple changes to have `activeTabButtonStyle` as an option to add custom different style for active button.

### Test plan

Inject style for custom active tab and see if style changes for active tab.

For example:

```Javascript
const TabNavigator = createBottomTabNavigator(
  {
    Home: HomePage
    Calendar: CalendarPage,
    Diary: DiaryPage,
    Settings: SettingsPage
  },
  {
    showLabel: false,
    activeTabButtonStyle: {
      borderTopColor: sicklyYellow,
      borderTopWidth: 2
    }
  }
)
```

This will result to custom active style as screenshot below:

Screen Shot 2562-07-14 at 17 15 20

### CODE REVIEW

File: src/views/BottomTabBar.js [hunk 1]
```diff
@@ -31,6 +31,7 @@ export type TabBarOptions = {
   showIcon: boolean,
   labelStyle: any,
   tabStyle: any,
+  activeTabButtonStyle?: any,
```
Comment (satya164) [inline src/views/BottomTabBar.js]: Rename this to `activeTabStyle` to match `tabStyle`

```suggestion
  activeTabButtonStyle?: any,
```

File: src/views/BottomTabBar.js [hunk 2]
```diff
@@ -390,6 +391,7 @@ class TabBarBottom extends React.Component<Props, State> {
                     ? styles.tabLandscape
                     : styles.tabPortrait,
                   tabStyle,
+                  focused ? this.props.activeTabButtonStyle : {},
```
Comment (satya164) [inline src/views/BottomTabBar.js]:
```suggestion
                  focused ? this.props.activeTabButtonStyle : null,
```

File: src/views/BottomTabBar.js (hunk 1 above)
Comment (chrisza4) [reply src/views/BottomTabBar.js]: `tabStyle` would set style of whole tab bar. This set style of only single tab button. Shall we name it differently?

File: src/views/BottomTabBar.js (hunk 2 above)
Comment (chrisza4) [reply src/views/BottomTabBar.js]: Thanks for suggestion here.

File: src/views/BottomTabBar.js (hunk 1 above)
Comment (satya164) [reply src/views/BottomTabBar.js]: `tabStyle` is for a single tab. It's used in the same place where the new prop is used

image

**Comment by chrisza4** [comment]
Ahh. CI failed. Let me fix this properly tonight.

**Comment by chrisza4** [comment]
@satya164 I changed according to the review. Sorry for the delay.

**Comment by jpacareu** [comment]
Sorry, what happened with this feature? I think is pretty useful

**Pull request closed without merging**